.gitignore

```diff
@@ -1,5 +1,6 @@
 *.pyc
 *.xml
 *.iml
+*.png
 
```

Comment:
do we want to ignore .c and .so files as well? do we need those to be pushed to the repo?

Reply:
If we don't push them, whoever downloads needs to compile it themselves before running optimization_time.py - so i can add directions in the readme for that. Or we can leave them and they won't have to do that.

Reply:
we can leave it then